internal/dto/env.go

```diff
@@ -2,23 +2,19 @@ package dto
 
 import (
 	"fmt"
-	"go.arcalot.io/log"
 	"os"
+
+	"go.arcalot.io/log"
 )
 
-func LookupEnvVar(key string, logger log.Logger) Verbose {
+func LookupEnvVar(registries string, key string, logger log.Logger) (string, error) {
```

Comment:
I think the variable name registries is confusing. It's the URL of a single registry.
It may help to add some documentation comments.

Reply:
I have changed the variable name to `registry_url` and have added documentation to the function. The variable name is passed in only to help with error message handling as to be more specific, but does not serve a purpose in the function itself of checking environment variables. Therefore, I did not add additional information there in the comments. Let me know if you would like another one liner that explains that.

Reply:
I only see the variable name change, and not the documentation.

Reply:
Yep. Looks like maybe your documentation commit didn't make it in, perhaps?

Reply:
Yeah. I am not sure how that happened but they were not part of the commit. I have added them.